app/audit/models/TaxYearOverviewResponseAuditModel.scala

```diff
@@ -44,25 +43,24 @@ case class TaxYearOverviewResponseAuditModel(mtdItUser: MtdItUser[_],
     ("taxableIncome", calculation.totalTaxableIncome) ++
     ("taxDue", calculation.totalIncomeTaxAndNicsDue)
 
-  private def getChargeType(chargeType: Option[String]): String = chargeType match {
-    case Some("ITSA- POA 1") => "Payment on account 1 of 2"
-    case Some("ITSA - POA 2") => "Payment on account 2 of 2"
-    case Some("TRM New Charge") | Some("TRM Amend Charge") => "Remaining balance"
-    case error => {
-      Logger.error(s"[TaxYearOverview][getChargeType] Missing or non-matching charge type: $error found")
-      "unknownCharge"
-    }
-  }
-
   private def paymentsJson(docDateDetail: DocumentDetailWithDueDate): JsObject = {
-    Json.obj("paymentType" -> getChargeType(docDateDetail.documentDetail.documentDescription),
-      "underReview" -> docDateDetail.dunningLock,
+    Json.obj(("paymentType", getChargeType(docDateDetail.documentDetail, false))) ++
+    Json.obj("underReview" -> docDateDetail.dunningLock,
       "status" -> docDateDetail.documentDetail.getChargePaidStatus) ++
       ("amount", docDateDetail.documentDetail.originalAmount) ++
       ("dueDate", docDateDetail.dueDate)
```

Comment:
Alternatively to previous suggestions the `++` syntax for optional field values could be used, like:
```suggestion
    Json.obj("underReview" -> docDateDetail.dunningLock,
      "status" -> docDateDetail.documentDetail.getChargePaidStatus) ++
      ("paymentType", getChargeType(docDateDetail.documentDetail)) ++
      ("amount", docDateDetail.documentDetail.originalAmount) ++
      ("dueDate", docDateDetail.dueDate)
```